modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/test/java/com/liferay/dynamic/data/mapping/form/builder/internal/converter/serializer/DefaultDDMFormRuleActionSerializerTest.java

```diff
@@ -98,6 +98,20 @@ public void testSerializeSetVisible() {
 		Assert.assertEquals("setVisible('field0', true)", result);
 	}
 
+	@Test
+	public void testSerializeWithEmptyTarget() {
+		DefaultDDMFormRuleAction defaultDDMFormRuleAction =
+			new DefaultDDMFormRuleAction("show", StringPool.BLANK);
+
+		DefaultDDMFormRuleActionSerializer defaultDDMFormRuleActionSerializer =
+			new DefaultDDMFormRuleActionSerializer(defaultDDMFormRuleAction);
+
+		String result = defaultDDMFormRuleActionSerializer.serialize(
+			_spiDDMFormRuleSerializerContext);
+
+		Assert.assertNull(result);
+	}
+
```

Comment:
@alinedoleron, this should be part of the last commit of SF, at this moment, the SF commit only has removed code. 😞